src/browser/components/MainPage.jsx

```diff
@@ -17,6 +16,7 @@ import DotsVerticalIcon from 'mdi-react/DotsVerticalIcon';
 import {ipcRenderer, remote, shell} from 'electron';
 
 import Utils from '../../utils/util';
+import url from '../../utils/url';
```

Comment:
we should probably use a different name to prevent people for misleading interpret one library as the other since `url` is a known node module and people might be used to it. this should also be applied to the rest of files.
```suggestion
import * as urlUtils from '../../utils/url';
```
if you prefer any other name that's fine too :D

Reply:
Alright, I have it named as `urlUtils` in other files, so I'll change them all to be the that. However, that existing code does `import urlUtils from 'utils/url';` rather than `import * as urlUtils from 'utils/url'`, is one preferred over the other?

Reply:
no, that's fine as long as it imports :D

Reply:
Should be good now!